boards/arm/sparkfun_thing_plus_nrf9160/sparkfun_thing_plus_nrf9160_defconfig

```diff
@@ -13,9 +13,6 @@ CONFIG_ARM_TRUSTZONE_M=y
 # Hardware stack protection
 CONFIG_HW_STACK_PROTECTION=y
 
-# enable GPIO
-CONFIG_GPIO=y
-
```

Comment:
The same should be done for the `_ns` target:
https://github.com/zephyrproject-rtos/zephyr/blob/05a65a4e7e73d83ac4713a51813a6e91d6a80c9e/boards/arm/sparkfun_thing_plus_nrf9160/sparkfun_thing_plus_nrf9160_ns_defconfig#L19-L20